readthedocs/proxito/middleware.py

```diff
@@ -196,6 +207,10 @@ def add_hsts_headers(self, request, response):
         * For custom domains, check the HSTS values on the Domain object.
           The domain object should be saved already in request.domain.
         """
+        if not request.is_secure():
+            # Only set the HSTS header if the request is over HTTPS
+            return response
```

Comment:
Good catch. Probably want a test for this 👍

Reply:
I found it because I broke a test that someone's else wrote 😉